ocaml/xapi/xapi_http.ml

```diff
@@ -161,10 +161,14 @@ let assert_credentials_ok realm ?(http_action = realm) ?(fn = Rbac.nofn)
         raise (Failure (Printf.sprintf "Unknown authorization header: %s" x))
     | None, None, None ->
         debug "No header credentials during http connection to %s" realm ;
+        (* The connection may have been authenticated using a client cert.
+         * If so, then following call confirms this. *)
         let session_id =
           try create_session_for_client_cert req ic
           with _ -> raise (Http.Unauthorised realm)
         in
+        debug "Created a session %s for a HTTP(s) request"
+          (Context.trackid_of_session (Some session_id)) ;
```

Comment:
The trackid may already get logged from elsewhere. I wanted to explicitly call out the client cert aspect here.